code/modules/antagonists/fugitive/hunters/hunter.dm

```diff
@@ -124,7 +124,7 @@
 	else//from here on out, hunters lost because they did not capture any fugitive dead or alive. there are different levels of getting beat though:
 		if(!fugitives_dead)//all fugitives survived
 			return FUGITIVE_RESULT_MAJOR_FUGITIVE
-		else if(fugitives_dead < fugitives_counted)//at least ANY fugitive lived
+		else if(fugitives_dead.len < fugitives_counted.len)//at least ANY fugitive lived
```

Comment:
```suggestion
		else if(length(fugitives_dead) < length(fugitives_counted))//at least ANY fugitive lived
```
`list.len` should never be used unless absolutely needed (and it is not needed for comparisons)